fact-bounty-client/src/pages/main/components/MainLayout.js

```diff
@@ -10,7 +10,7 @@ class MainLayout extends Component {
 
 	render() {
 		return (
-			<div className="mainLayout">
+			<div className="mainLayout" style={{paddingTop: '70px'}}>
```

Comment:
Remove inline css and take the padding style to mainLayout class

Reply:
@root5533  done, please review 😃